scripts/format-markdown.sh

```diff
@@ -2,7 +2,28 @@
 
 SRCDIR="$(dirname "${BASH_SOURCE[0]}")" # check on Windows
 
-for filename in $*; do
+CONFIG=""
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --config)
+      CONFIG="$2"
+      shift # Remove --config from processing
+      shift # Remove the value from processing
+      ;;
+    *)
+      FILES+="$1 "
+      shift # Move to the next argument
+      ;;
+  esac
+done
+
+if [ -z "$CONFIG" ]; then
+  echo "Error: --config parameter is required."
+  exit 1
+fi
+
+for filename in $FILES; do
```

Comment:
Not necessary, same rules for all files

Reply:
I'm not sure what this is used for but I definitely don't see that we need this script, can someone explain what the problem was that this was solving? Let's try again!